backend/app/api/endpoints/adapter/subscription_follows.py

```
@@ -0,0 +1,343 @@
+logger = logging.getLogger(__name__)
+
+router = APIRouter()
+
+
+# ========== Follow/Unfollow Endpoints ==========
+
+
+@router.post("/{subscription_id}/follow", status_code=status.HTTP_200_OK)
+def follow_subscription(
+    subscription_id: int,
+    db: Session = Depends(get_db),
```

Ruff (0.14.13)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)